examples/inflate-helm-chart-crds/README.md

```diff
@@ -0,0 +1,65 @@
+# inflate-helm-chart: Remote Chart with CRDs
+
+### Overview
+
+This example demonstrates how to imperatively invoke the `inflate-helm-chart`
+function to inflate a helm chart that contains CRDs in the templated output.
+
+### Function invocation
+
+First, let's inflate a terraform chart without CRDs:
+
+```shell
+$ kpt fn eval --image gcr.io/kpt-fn/inflate-helm-chart:unstable --network -- \
+name=terraform \
+repo=https://helm.releases.hashicorp.com \
+version=1.0.0 \
+releaseName=terraforming-mars 
+```
+
+You should have several files in your local filesystem. You can run the
```

Comment:
`You should see rendered chart resources in your local filesystem` ?

Reply:
Done